tests/test-rest-comments-controller.php

```diff
@@ -1048,7 +1048,6 @@ public function test_create_comment_private_post_invalide_permission() {
 	}
 
 	public function test_create_item_duplicate() {
-		$this->markTestSkipped( 'Needs to be revisited after wp_die handling is added' );
```

Comment:
@websupporter Because the test will only pass for WP trunk, you need a conditional here.
```
if ( version_compare( $wp_version, '4.7-alpha', '<' ) ) {
            return $this->markTestSkipped( 'WordPress version not supported.' );
}
```